src/Account/index.jsx

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
-import { View, Text } from 'react-native';
-import commonStyles from '../common/styles';
-
-export default () => (
-  <View style={commonStyles.container}>
-    <Text>This is the account</Text>
-  </View>
-);
+import { View, Text, Button } from 'react-native';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+import { logOut } from '../redux/account/actionCreator';
+import styles from './styles';
+
+const Account = () => {
+  const { accessToken } = useSelector((state) => state.user);
+  const { username } = useSelector((state) => state.user);
+  const dispatch = useDispatch();
+  const { push } = useNavigation();
+
+  const handleLogOut = () => {
+    dispatch(logOut());
+  };
+
+  return (
+    <View style={styles.container}>
+      {accessToken
+        ? (
+          <View>
+            <Text style={{ fontSize: 20 }}>
+              {`Welcome to our app, ${username}`}
+            </Text>
+            <Button style={styles.logOutButton} title="logout" onPress={handleLogOut} />
+          </View>
+        ) : (
+          <View>
+            <Text>Please Login</Text>
+            <Button title="Login" onPress={() => push('Login')} />
```

Comment:
please be consequent and use either capitalized titles or  lowercased, but do not mix them
```suggestion
            <Button title="login" onPress={() => push('Login')} />
```